core/src/main/java/org/mule/runtime/core/api/streaming/DefaultStreamingManager.java

```diff
@@ -124,6 +129,19 @@ public CursorProvider manage(CursorProvider provider, CoreEvent creatorEvent) {
    * {@inheritDoc}
    */
   @Override
+  public void manage(InputStream stream, CoreEvent creatorEvent) {
```

Comment:
Why do you need to pass event, doesn't EventContext suffice?

Reply:
i pass the event mainly for consistency with the other manage method which is already there and receives the full event. A very secondary `future proofing` point could also be made. By passing the entire event the API is still good if we ever need something else than the context. But that argument pales compared to the consistency one

Reply:
ok. thought it was that, just checking,